src/style.css

```diff
@@ -1,3 +1,124 @@
+/* Stylowanie wszystkich divów */
+/* div,
+nav {
+  border: 2px solid;
+} */
+
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.MoyoHeader {
+  display: flex;
+  width: 100%;
+  height: 60px;
+
+  justify-content: space-between;
+  align-items: center;
+}
+
+.MoyoHeader__nav__links a {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
```

Comment:
it would be better to add another class on `<a>` tag, instead of nesting selectors